python/ray/utils.py

```diff
@@ -11,6 +11,52 @@
 
 import ray.local_scheduler
 
+ERROR_KEY_PREFIX = b"Error:"
+
+
+def _random_string():
+    return np.random.bytes(20)
+
+
+def push_error_to_driver(redis_client, driver_id, error_type, message,
```

Comment:
Let's unify these two methods into one and push to all drivers if driver_id = "*" or driver_id = None. This resolves the method parameter inconsistencies and code duplication.